drivers/mbox/mbox_nrf_vevif_event_tx.c

```diff
@@ -39,6 +39,13 @@ static int vevif_event_tx_send(const struct device *dev, uint32_t id, const stru
 
 	nrf_vpr_csr_vevif_events_trigger(BIT(id));
 
+#if defined(CONFIG_MBOX_NRF_VEVIF_EVENT_USE_54L_ERRATA_16)
```

Comment:
MDK errata symbols/functions could be used here as well as on rx side: https://github.com/zephyrproject-rtos/hal_nordic/blob/master/nrfx/mdk/nrf54l_erratas.h#L52